charts/aws-calico/templates/deployment.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "{{ include "aws-calico.fullname" . }}-typha"
+  labels:
+    app.kubernetes.io/name: "{{ include "aws-calico.fullname" . }}-typha"
+{{ include "aws-calico.labels" . | indent 4 }}
+spec:
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "{{ include "aws-calico.fullname" . }}-typha"
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: "{{ include "aws-calico.fullname" . }}-typha"
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: 'true'
+        {{- with .Values.calico.typha.podAnnotations }}
+        annotations: {{- toYaml . | nindent 10 }}
+        {{- end }}
+    spec:
+      priorityClassName: system-cluster-critical
+      nodeSelector:
+        {{- toYaml .Values.calico.typha.nodeSelector | nindent 8 }}
+      tolerations:
+        # Mark the pod as a critical add-on for rescheduling.
+        - key: CriticalAddonsOnly
+          operator: Exists
+        {{- if .Values.calico.typha.tolerations }}
+        {{- toYaml .Values.calico.typha.tolerations | nindent 8 }}
+        {{- end }}
+      hostNetwork: true
+      serviceAccountName: "{{ include "aws-calico.serviceAccountName" . }}-node"
+      # fsGroup allows using projected serviceaccount tokens as described here kubernetes/kubernetes#82573
+      securityContext:
+        fsGroup: 65534
+      containers:
+        - image: "{{ .Values.calico.typha.image }}:{{ .Values.calico.tag }}"
+          name: calico-typha
+          ports:
+            - containerPort: 5473
```

Comment:
is this port not configurable?  seems like it should be especially since this pod runs with `hostNetworking: true`

Reply:
Yes, will make it configurable.